yarn.lock

```diff
@@ -2671,6 +2671,30 @@
   dependencies:
     tslib "^2.4.0"
 
+"@synthql/backend@0.38.4":
```

Comment:
why did this pop up here?

Reply:
Added the `backend` package to be able to use the QueryEngine in the `react` tests